### PR TITLE
Add Prymont + Add `eth2_network_name` field to deposit_data*.json file

### PR DESCRIPTION
**Pull request by hwwhww**
- Add `PyrmontSetting`
- Add `eth2_network_name` field to deposit_data*.json file

### CODE REVIEW

**Review by CarlBeek — COMMENTED** [review]
Looks good to me 👍

**Review by CarlBeek — APPROVED** [review]

**Comment by protolambda** [comment]
Hold off, I am going to redo pyrmont deposits, so the contract changes. To fix the fork version.

File: eth2deposit/settings.py [hunk 1]
```diff
@@ -5,38 +5,45 @@
...
-MedallaSetting = BaseChainSetting(GENESIS_FORK_VERSION=bytes.fromhex('00000001'))
+MedallaSetting = BaseChainSetting(ETH2_NETWORK_NAME=MEDALLA, GENESIS_FORK_VERSION=bytes.fromhex('00000001'))
 # Eth2 "dress rehearsal" testnet (spec v0.12.3)
-SpadinaSetting = BaseChainSetting(GENESIS_FORK_VERSION=bytes.fromhex('00000002'))
+SpadinaSetting = BaseChainSetting(ETH2_NETWORK_NAME=SPADINA, GENESIS_FORK_VERSION=bytes.fromhex('00000002'))
 # Eth2 "dress rehearsal" testnet (spec v0.12.3)
-ZinkenSetting = BaseChainSetting(GENESIS_FORK_VERSION=bytes.fromhex('00000003'))
+ZinkenSetting = BaseChainSetting(ETH2_NETWORK_NAME=ZINKEN, GENESIS_FORK_VERSION=bytes.fromhex('00000003'))
+# Eth2 testnet (spec v1.0.0)
+PyrmontSetting = BaseChainSetting(ETH2_NETWORK_NAME=PYRMONT, GENESIS_FORK_VERSION=bytes.fromhex('00000000'))
```
Comment (hwwhww) [inline eth2deposit/settings.py:34]:
```suggestion
# Eth2 pre-launch testnet (spec v1.0.0)
PyrmontSetting = BaseChainSetting(ETH2_NETWORK_NAME=PYRMONT, GENESIS_FORK_VERSION=bytes.fromhex('00002009'))
```

Comment (protolambda) [reply eth2deposit/settings.py]: Confirm :+1: 

Other pyrmont testnet details can be found here: https://github.com/protolambda/pyrmont

**Pull request merged**